content/en/docs/concepts/services-networking/network-policies.md

```diff
@@ -31,7 +31,7 @@ Network policies are implemented by the [network plugin](/docs/concepts/extend-k
 
 By default, pods are non-isolated; they accept traffic from any source.
 
-Pods become isolated by having a NetworkPolicy that selects them. Once there is any NetworkPolicy in a namespace selecting a particular pod, that pod will reject any connections that are not allowed by any NetworkPolicy. (Other pods in the namespace that are not selected by any NetworkPolicy will continue to accept all traffic.)
+Pods become isolated by having a NetworkPolicy that selects them. Once there is any NetworkPolicy in a namespace selecting a particular pod, that pod will reject any connections that are not allowed by that NetworkPolicy. (Other pods in the namespace that are not selected by any NetworkPolicy will continue to accept all traffic.)
```

Comment:
if anything it should probably be "those network policies".

"that NetworkPolicy" is singular, and what this is trying to say is that any connections which are not allowed by at least one NetworkPolicy will be rejected.


Probably wrong to say "that pod will reject" as well.

How about this:

> Pods become isolated as soon as a NetworkPolicy's spec.PodSelector selects them. Once there is any NetworkPolicy selecting a given pod, connections to and from that pod will fail unless they are allowed by at least one NetworkPolicy.

Reply:
I think @caseydavenport  suggestion makes sense.

Reply:
It probably needs a little more clarification, now that I read it a second time.

The main thing being that ingress/egress are independent of each other.

> Pods become isolated as soon as a NetworkPolicy's spec.PodSelector selects them. Once there is any Ingress NetworkPolicy selecting a given pod, connections to that pod will fail unless they are allowed by at least one ingress rule. Similarly, once any Egress NetworkPolicy selects a given pod, connections from that pod will fail unless they are allowed by at least one egress rule.